### PR TITLE
Add zuul CI integration

### LINKED ISSUE #266: Add zuul CI integration
Base set of checks should include lints:
- pep8/pylint
- yamllint
- terraform validate

### PR DESCRIPTION
**Pull request by outcatcher**
Move to zuul from travis

Add linters for python and ansible

Update playbooks requirements

Resolve #266

### CODE REVIEW

File: playbooks/files/requirements.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-ocomone >= 0.3.3
-csm-test-utils >= 0.2.5
-boto3 == 1.10.13
-botocore == 1.13.13
-cryptography == 3.2
+ocomone== 0.4.3
```
Comment (lego963) [inline playbooks/files/requirements.txt]: Remove space after ==

**Review by lego963 — CHANGES_REQUESTED** [review]
Change ansible version in readme pls

**Comment by outcatcher** [comment]
recheck

**Review by anton-sidelnikov — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by lego963 — DISMISSED** [review]

**Comment by outcatcher** [comment]
recheck

**Review by anton-sidelnikov — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Comment by lego963** [comment]
regate

**Pull request merged**